fab/environments.yml

```diff
@@ -110,19 +110,15 @@ enikshay:
       GroupToUserPillow:
         num_processes: 1
       kafka-ucr-main:
-        num_processes: 4
+        num_processes: 2
```

Comment:
This could probably be 1 actually. UCRs aren't processed here since you guys use the async queue. This just puts them in the queue